pkg/manifest/context.go

```diff
@@ -25,7 +27,8 @@ type SMTP struct {
 
 type Context struct {
 	Bundles       []*Bundle
-	Buckets       []string
+	Buckets       sets.String
+	Domains       sets.String
```

Comment:
Changing this from a `[]string` to a map is causing errors with existing installations. We should use `[]string` for these, since a map with empty values might confuse users and be more error prone.